### PR TITLE
JobDSL

### PR DESCRIPTION
**Pull request by sdorra**
Support custom JobDSL syntax to simplify the creation of SCM-Manager Multibranch Pipelines with the JobDSL.

### CODE REVIEW

File: docs/de/index.md [hunk 1]
```diff
@@ -39,6 +39,51 @@ werden muss.
...
+multibranchPipelineJob('heart-of-gold') {
+  branchSources {
+    scmManager {
+      id('spaceships/heart-of-gold')
+      serverUrl('https://scm.hitchhiker.com')
+      credentialsId('my-secret-id')
+      repository('spaceships/heart-of-gold')
+      discoverBranches(true)
+      discoverPullRequest(true)
+      discoverTags(true)
```
Comment (eheimbuch) [inline docs/de/index.md:57]: Isn't default for `discoverTags` = false

File: README.md [hunk 2]
```diff
@@ -34,6 +34,49 @@ plugin will then load a list of all available repositories you can select from.
...
+multibranchPipelineJob('heart-of-gold') {
+  branchSources {
+    scmManager {
+      id('spaceships/heart-of-gold')
+      serverUrl('https://scm.hitchhiker.com')
+      credentialsId('my-secret-id')
+      repository('spaceships/heart-of-gold')
+      discoverBranches(true)
+      discoverPullRequest(true)
+      discoverTags(true)
```
Comment (eheimbuch) [inline README.md:51]: Isn't default for discoverTags = false

**Review by eheimbuch — CHANGES_REQUESTED** [review]

File: docs/de/index.md (hunk 1 above)
Comment (sdorra) [reply docs/de/index.md]: You are right.

File: README.md (hunk 2 above)
Comment (sdorra) [reply README.md]: jo

**Pull request merged**